ClientApp/src/components/superadmin/kommune/KommuneInformation.tsx

```diff
@@ -37,6 +37,23 @@ const KommuneInformation: React.FC<IKommuneInformation> = ({ accessToken, locati
   };
   useEffect(fetchInformation, [location]);
 
+  const deleteLocation = () => {
+    apiservice
+      .post("Cms/deleteSingle", {
+        ContentType: "Kommune",
+        Index: location,
+        Question: "",
+        Info: "",
```

Comment:
Question and Info can be removed here, no reason to send empty strings to back end as long as the corresponding properties are nullable in the dto class.